tools/config-generator/main.go

```diff
@@ -853,41 +796,14 @@ func recursiveSBL(repoName string, data interface{}, generateOneJob func(data in
 // executeJobTemplateWrapper takes in consideration of repo settings, decides how many variants of the
 // same job needs to be generated and generates them.
 func executeJobTemplateWrapper(repoName string, data interface{}, generateOneJob func(data interface{})) {
-	var sbs []specialBranchLogic
-
 	switch data.(type) {
```

Comment:
Is this switch necessary with the new logic? It seems for all cases we are invoking `generateOneJob(data)`

Reply:
You are right, it's not needed anymore. But I'm a bit worried that we'll need the `specialBranchLogic` again in the future, so I feel we should not clean up too much of the code.

cc @chaodaiG for opinions since he was the initial author for the logic here.

Reply:
I have the same feeling with Chi, would suggest we keep the logic for a little longer.

I have another question though: it's nice to converge master and release branch jobs, but would this mean that previous release branch with no go.mod will use go1.13?

Reply:
That is true. But nothing regresses as the infra is only supposed to support the last 4 releases, and all of them are already using Go 1.13.

Reply:
Ack

Reply:
I'm fine keeping the code, especially if it's going to be restored in the near future. But I would want to hear we will need it in the short-term (during the next few weeks). Otherwise, I will consider removing it for health purposes. We will always have the history of the file to get it back if we need it.

Reply:
Makes sense. I have cleaned up more of the code.